spec/configs/stanford_organizations_to_sparql_statements_spec.rb

```diff
@@ -0,0 +1,322 @@
+                          'education-and-recreation'],
+                                 Rialto::Etl::Vocabs::SKOS['prefLabel'],
+                                 'Physical Education and Recreational Lounging'])
+                       .true?
+        expect(result).to be true
+
+        # Changes org code
+        result = client.ask
+                       .from(Rialto::Etl::NamedGraphs::STANFORD_ORGANIZATIONS_GRAPH)
+                       .whether([Rialto::Etl::Vocabs::RIALTO_ORGANIZATIONS['department-of-athletics-physical-education'\
+                          '-and-recreation'],
+                                 Rialto::Etl::Vocabs::DCTERMS.identifier,
+                                 'LVTK'])
+                       .whether([Rialto::Etl::Vocabs::RIALTO_ORGANIZATIONS['department-of-athletics-physical-'\
+                          'education-and-recreation'],
+                                 Rialto::Etl::Vocabs::DCTERMS.identifier,
+                                 'LVRC'])
+                       .true?
+        expect(result).to be false
+        result = client.ask
+                       .from(Rialto::Etl::NamedGraphs::STANFORD_ORGANIZATIONS_GRAPH)
+                       .whether([Rialto::Etl::Vocabs::RIALTO_ORGANIZATIONS['department-of-athletics-physical-'\
+                          'education-and-recreation'],
+                                 Rialto::Etl::Vocabs::DCTERMS.identifier,
+                                 'LVTK'])
+                       .whether([Rialto::Etl::Vocabs::RIALTO_ORGANIZATIONS['department-of-athletics-physical-'\
+                          'education-and-recreation'],
+                                 Rialto::Etl::Vocabs::DCTERMS.identifier,
+                                 'LVRX'])
+                       .true?
+        expect(result).to be true
+
+        # Changes parent
+        result = client.ask
+                       .from(Rialto::Etl::NamedGraphs::STANFORD_ORGANIZATIONS_GRAPH)
+                       .whether([Rialto::Etl::Vocabs::RIALTO_ORGANIZATIONS['department-of-athletics-physical-'\
+                          'education-and-recreation'],
+                                 Rialto::Etl::Vocabs::OBO['BFO_0000050'],
+                                 Rialto::Etl::Vocabs::RIALTO_ORGANIZATIONS['stanford']])
+                       .true?
+        expect(result).to be false
+        result = client.ask
+                       .from(Rialto::Etl::NamedGraphs::STANFORD_ORGANIZATIONS_GRAPH)
+                       .whether([Rialto::Etl::Vocabs::RIALTO_ORGANIZATIONS['department-of-athletics-physical-'\
+                          'education-and-recreation'],
+                                 Rialto::Etl::Vocabs::OBO['BFO_0000050'],
+                                 Rialto::Etl::Vocabs::RIALTO_ORGANIZATIONS['amherst']])
+                       .true?
+        expect(result).to be true
+
+        # TODO: Change parent
```

Comment:
What does this mean?

Reply:
It means a test needs to be written for changing the parent. But what I really meant was a test needs to be written for changing children (once support for org children is implemented). Fixing.